packages/core/src/app/payment/hostedCreditCard/withHostedCreditCardFieldset.spec.tsx

```diff
@@ -12,7 +12,6 @@ import { act } from 'react-dom/test-utils';
 
 import {
     CardInstrumentFieldsetValues,
-    HostedCreditCardFieldsetValues,
 } from '@bigcommerce/checkout/payment-integration-api';
```

Comment:
`HostedCreditCardFieldsetValues` does not exist in the API. So import it from the core on line 26.
It's an error picked up by IDE, not exactly part of the work.